src/plugins/arkeo/Arkeo.tsx

```diff
@@ -0,0 +1,145 @@
+import { ArrowForwardIcon } from '@chakra-ui/icons'
+import { Box, Button, Flex, Grid, Heading, Image, Link, useColorModeValue } from '@chakra-ui/react'
+import type { DefiType } from 'features/defi/contexts/DefiManagerProvider/DefiCommon'
+import { useTranslate } from 'react-polyglot'
+import ArkeoBg from 'assets/arkeo-bg.jpg'
+import NodeImage from 'assets/node.svg'
+import { Main } from 'components/Layout/Main'
+import { SEO } from 'components/Layout/Seo'
+import { RawText, Text } from 'components/Text'
+import { WalletActions } from 'context/WalletProvider/actions'
+import { KeyManager } from 'context/WalletProvider/KeyManager'
+import { useWallet } from 'hooks/useWallet/useWallet'
+import { isMobile as isMobileApp } from 'lib/globals'
+import type { OpportunityId } from 'state/slices/opportunitiesSlice/types'
+
+import { FoxTokenHolders } from './FoxTokenHolders'
+import { LpCards } from './LpCards'
+import { StakingCards } from './StakingCards'
+
+type OpportunityReturn = {
+  [k in DefiType]: OpportunityId[]
+}
+
+const opportunities: OpportunityReturn = {
+  staking: [
+    'eip155:1/erc20:0xee77aa3fd23bbebaf94386dd44b548e9a785ea4b',
```

Comment:
`FOXY_STAKING_CONTRACT`

Reply:
This one doesn't seem to be exported anywhere...

Reply:
Yeah, we would need to export `constants` from lib's `packages/unchained-client/src/evm/ethereum/parser/index.ts` - fine re-declared in this module given the one-off use case